README.rst

```diff
@@ -20,25 +20,18 @@ Django-money
 A little Django app that uses ``py-moneyed`` to add support for Money
 fields in your models and forms.
 
-Fork of the Django support that was in
-http://code.google.com/p/python-money/
-
-This version adds tests, and comes with several critical bugfixes.
-
-Django versions supported: 1.11, 2.1, 2.2
-
-Python versions supported: 3.5, 3.6, 3.7
-
-PyPy versions supported: PyPy3 2.4
+* Django versions supported: 1.11, 2.1, 2.2, 3.0
+* Python versions supported: 3.5, 3.6, 3.7
+* PyPy versions supported: PyPy3 (and PyPy for Django 1.11)
```

Comment:
Hm, after dropping Python 2 support, it should mean that PyPy is not supported as well, since it is based on Python2.7, could you check it, please?

Reply:
I was looking at our tox.ini config for this... and I saw it mentioned... will check!

Reply:
The tox entry I was looking at wasn't active in Travis, so removed it